packages/web-components/fast-components/src/styles/patterns/button.ts

```diff
@@ -58,9 +60,7 @@ export const BaseButtonStyles = (
             font-family: ${bodyFont};
             outline: none;
             font-size: ${typeRampBaseFontSize};
-            line-height: ${typeRampBaseLineHeight};
-            height: calc(${heightNumber} * 1px);
-            min-width: calc(${heightNumber} * 1px);
+            line-height: ${typeRampBaseVisualHeight};
```

Comment:
Related to the comment above about single line height + leading. Since single line is most common, it may be easier to repurpose what line height means in the token and to have a derived multi-line height for the fewer cases that actually have multiple lines of text.